standby/src/lib.rs

```diff
@@ -164,7 +199,7 @@ impl Standby {
     /// This function must be called when events are received in order for
     /// futures returned by methods to fulfill.
     pub fn process(&self, event: &Event) {
-        log::trace!("Processing event: {:?}", event);
+        log::debug!("Processing event: {:?}", event);
```

Comment:
can this not end up giving an incredible amount of spam in the log if it is a larger bot?

Reply:
Fixed